### DISCUSSION TITLE
What's the buggyness from running a dedicated server?

### DISCUSSION THREAD

**Discussion by rubin110** [wolveix/satisfactory-server#278] [Q&A] [answered]
Hello! I've never played Satisfactory before, however I have some friends who really want to play it with me. I found your docker image (thank you) and started reading in order to get a server setup, however I saw this warning:

> If the server feels too buggy for you, you can try the [saveshare](https://github.com/wolveix/satisfactory-server/blob/main/saveshare/README.md) instead (which relies on client-hosting).

Is there a description of what this buggyness is so I can discuss it with my friends and figure out which way I should be setting up this server?

Thanks!

**Comment by wolveix** [comment, marked as answer]
Hey! That's just a generic message that's been there for years. Dedicated servers used to be buggy, with 1.0 they're no longer in beta and should be much more stable!

**Comment by rubin110** [comment]
Thanks for the info!
